src/Raven.Studio/wwwroot/Content/css/pages/documents-edit.less

```diff
@@ -219,8 +219,9 @@
         }
     }
 
-    #copy-document-id-icon {
+    #copy-document-id {
         visibility: hidden;
+        color: @gray-light;
```

Comment:
I think there is no purpose of having that ?

Reply:
This was added because when **'a'** is used instead of **'span**' then the default color is **red**

Reply:
ok, thanks